src/test/java/com/epam/steps/admin/CreateTeacherSteps.java

```diff
@@ -2,19 +2,21 @@
 
 import com.epam.pages.main.AdminPage;
 import com.epam.pages.popup.CreatePopup;
+import com.epam.steps.BaseSteps;
 import io.cucumber.java.en.And;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class CreateTeacherSteps {
+public class CreateTeacherSteps extends BaseSteps {
     private final AdminPage adminPage = new AdminPage();
     private final CreatePopup createPopup = new CreatePopup();
-
-    @Then("See all elements are present")
-    public void seeAllElementsArePresent() {
-        assertThat(adminPage.checkAllElementsArePresent()).isTrue();
+    @Then("See all elements are present on admin page")
+    public void seeAllElementsArePresentOnAdminPage() {
+        assertThat(adminPage.checkAllElementsArePresent())
+                .withFailMessage("Elements in admin page are not present")
+                .isTrue();
     }
 
     @When("Select {} section")
```

Comment:
In my opinion, this step should't be in this class, because this class is for teacher creation, We can create CommonSteps class, and move this method to that class, because this step is used for creating teacher, parent, academic class, student, etc.

Reply:
this method was mowed to UICheckSteps